### DISCUSSION TITLE
Does not support latest version of Web3Modal Sign V2. Your Project ID is approved and Explorer is an optional step if you would like to have your project marketed on our homepage so that you can get a wider outreach and help users find your dapp. Docs to help you migrate to Sign V2: https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/dapps/dapp-checklist

### DISCUSSION THREAD

**Discussion by 1648045459** [WalletConnect/walletconnect-monorepo#3374] [Developer Support]
version:
 "@walletconnect/encoding": "^1.0.1",
    "@walletconnect/qrcode-modal": "^1.7.8",
    "@walletconnect/sign-client": "^2.0.0-rc.4",
    "@walletconnect/types": "^2.0.0-rc.4",
    "@walletconnect/utils": "^2.0.0-rc.4",

Relay Server:  https://relay.walletconnect.com

chainId:  eip155:137

When applying for a project ID in https://cloud.walletconnect.com/app, why does the following error appear and the application fails:

Changes requested

Your update has been rejected. Reason:

""Does not support latest version of Web3Modal Sign V2. Your Project ID is approved and Explorer is an optional step if you would like to have your project marketed on our homepage so that you can get a wider outreach and help users find your dapp. Docs to help you migrate to Sign V2: https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/dapps/dapp-checklist""

**Comment by glitch-txs** [comment]
Could you try updating to the latest version, also do you have a public repo to check this?
Have you tried creating a new project ID?
